<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Customlable" name="label_img">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lable_fps">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Current FPS: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_hw">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Hardware version: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_fw">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Firmware version: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_host">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Host: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lable_ip">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>75</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>IP: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_corners">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>175</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Corners</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_roi">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Roi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switch_mode">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>305</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fps_chart">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>FPS Chart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>560</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_gain">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>450</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="set_gain">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>450</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>435</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Gain</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_threshold">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>510</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>490</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Threshold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_threshold">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>510</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="roi_width">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>x: </string>
    </property>
   </widget>
   <widget class="QLabel" name="roi_height">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>150</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>30</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Top Left</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>50</y>
      <width>20</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>x: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>80</y>
      <width>20</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>y: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>100</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Bottom Right</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>20</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>x: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>150</y>
      <width>20</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>y: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="topleft_x">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>50</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="topleft_y">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bottomright_x">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>120</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="bottomright_y">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="set_roi">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set ROI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_roi">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_roi">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>260</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Customlable</class>
   <extends>QLabel</extends>
   <header>customlable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
